web/web_types/types.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web_types
+
+import (
+	"net/url"
+	"time"
+)
+
+type RuntimeInfo struct {
+	Birth               time.Time          `json:"startTime"`
+	CWD                 string             `json:"cwd"`
+	Version             *PrometheusVersion `json:"version"`
+	Alertmanagers       []*url.URL         `json:"alertmanagers"`
+	GoroutineCount      int                `json:"goroutineCnt"`
```

Comment:
```suggestion
	GoroutineCount      int                `json:"goroutineCNT"`
```

In golang acronyms are written in _all-caps_.

Reply:
We could also follow other standards. What do others think?